web-app/src/pages/Admin/index.jsx

```diff
@@ -34,6 +35,7 @@ export const Admin = props => (
           path="/admin/clients/"
           render={() => <AllClients history={props.history} />}
         />
+        <Route path="/admin/teamOverview" component={TeamOverview} />
```

Comment:
A small thing, but try clean up this URL to be similar to the others. Something like:
`/admin/overview/team`, or even `/admin/overview/`